content/content/Getting Started/overview.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Overview"
+date: 2021-11-16T12:08:47-05:00
+draft: false
+---
+
+# What is crane?
```

Comment:
See comments above on "_index.md", since the content is the same.

Reply:
should probably avoid having two identical pages...right? Seems like an easy thing for us to lose track of and get out of sync.